app/views/classifications/_document_list_from_rummager.html.erb

```diff
@@ -0,0 +1,34 @@
+<%
+   heading ||= type.to_s.humanize
+   filter_option = (heading == "Publications") ? "all" : heading.downcase
+%>
+<section id="<%= heading.downcase %>" class="document-block">
+  <h1 class="label"><%= heading %></h1>
+  <div class="content">
+    <%= render partial: "shared/list_description_for_rummager_list", locals: { heading: heading, editions: documents, documents_count: documents_count, type: type } %>
+    <p class="see-all">
+    <%-
+      see_more_url = public_send(
+        "#{type}_filter_path",
+        @classification,
+        publication_filter_option: filter_option
+      )
+    %>
+    <%=
+      link_to(
+        "See all #{heading.downcase}",
+        see_more_url,
+        data: {
```

Comment:
For our data-tracking to take effect, we normally have to add `data-module="track-click"` to the element itself or a parent. Has this been included anywhere? I normally test the analytics using something like [Google Analytics Debugger](https://chrome.google.com/webstore/detail/google-analytics-debugger/jnkmfdileelhofjcijamephohjechhna) to double check the tracking events are firing.

Reply:
This is included in the same place as the existing `_document_list`, so should behave in the same way (if the tracking on that template is broken this will be too).